client/src/extension.ts

```diff
@@ -0,0 +1,76 @@
+import * as path from 'path';
+import { workspace, ExtensionContext, commands, languages, Disposable } from 'vscode';
+
+import {
+    LanguageClient,
+    LanguageClientOptions,
+    ServerOptions,
+    TransportKind,
+} from 'vscode-languageclient';
+import OMTLinkProvider from './omtLinkProvider';
+
+let client: LanguageClient;
+
+export function activate(context: ExtensionContext) {
+    // The server is implemented in node
+    let serverModule = context.asAbsolutePath(
+        path.join('server', 'out', 'server.js')
+    );
+    // The debug options for the server
+    // --inspect=6009: runs the server in Node's Inspector mode so VS Code can attach to the server for debugging
+    let debugOptions = { execArgv: ['--nolazy', '--inspect=6009'] };
+
+    // If the extension is launched in debug mode then the debug server options are used
+    // Otherwise the run options are used
+    let serverOptions: ServerOptions = {
+        run: { module: serverModule, transport: TransportKind.ipc },
+        debug: {
+            module: serverModule,
+            transport: TransportKind.ipc,
+            options: debugOptions
+        }
+    };
+
+    // Options to control the language client
+    let clientOptions: LanguageClientOptions = {
+        // Register the server for OMT and ODT documents
+        documentSelector: [
+            { scheme: 'file', language: 'omt' },
+            { scheme: 'file', language: 'odt' },
+        ],
+        synchronize: {
+            // Notify the server about file changes to '.clientrc files contained in the workspace
+            fileEvents: workspace.createFileSystemWatcher('**/.clientrc')
```

Comment:
I have been looking into this but cannot find any information on the .clientrc files. apparently there can be more and the example from microsoft states it like this. Other examples use it the same way but I expect that may be because they copied it from that same example. 

- [ ] research what clientrc files do
- [ ] find out if the pattern can be changed or removed

Reply:
I think this is about the comment, which has .clientrc quoted on only one side. The comment is placed one line lower than it should have been ;)